script.go

```diff
@@ -385,7 +385,8 @@ func (p *Pipe) Exec(cmdLine string) *Pipe {
 		cmd.Stdin = r
 		cmd.Stdout = w
 		cmd.Stderr = w
-		if p.stderr != nil {
+		pipeStderr := p.getStderr()
+		if pipeStderr != nil {
 			cmd.Stderr = p.stderr
 		}
```

Comment:
I have a problem with the logic here

getStderr returns pipeStdErr


```suggestion
		pipeStderr := p.getStderr()
		if pipeStderr != nil {
 			cmd.Stderr = pipeStderr
 		}
```

While the code is somehow similar, even if there could remain some races, because we are accessing the p.stderr directly, no ?

Reply:
If not, the method and signature could be updated, because there is no need to return it, so,it's not a getter

Reply:
@ccoVeille this was an oversight! thanks for catching - fixed now